Differential/grace-apollos-app
# Add yarn setup and nuke script

This adds a `yarn setup` script that installs root, api, and client packages and pods instead of having to `cd` into each folder and `yarn install`. Also, adds our `yarn nuke` script to kill all `node_modules`

TO TEST:
Running `yarn setup` should install all packages and pods for the entire app.
Running `yarn nuke` should remove all `node_modules`
